sunpy/coordinates/screens.py

```diff
@@ -170,15 +174,15 @@ class PlanarScreen(BaseScreen):
 
     @u.quantity_input
     def __init__(self, vantage_point, distance_from_center: u.m=0*u.m, **kwargs):
```

Comment:
```suggestion
     def __init__(self, vantage_point, *, distance_from_center: u.m=0*u.m, **kwargs):
```
Make keyword-only like `SphericalScreen`